data/human/intro missions.txt

```diff
@@ -423,14 +507,14 @@ mission "Intro [3 Interceptor]"
 	description "Drop James off on <destination>, where he has a spot reserved in a retirement home."
 	blocked `When you meet up with James in the spaceport, he says, "Hi there, <first>! I found you another mission, but it looks like you'll need <capacity> in order to take it on. So, you'll either have to cancel another mission, or come back here later to continue this one. Sorry!"`
 	passengers 1
-	source "Ingot"
+	source "New Sahara"
 	destination "Hestia"
 	to offer
-		has "Intro [2 Interceptor]: done"
+		has "Intro [2.1 Interceptor]: done"
 	
 	on offer
 		conversation
-			`You find James sitting at the spaceport's tiny bar, sipping a beer while listening to the steady patter of micrometeorites against the roof. "Working here," says James, "is one of the few jobs in the galaxy more dangerous than fighting pirates. And worse, because if a big meteorite decides to strike this station, there's basically nothing they can do about it except hope they see it coming. Care for a drink?"`
+			`You find James sitting at the spaceport's tiny bar, sipping a beer while watching the large displays that show the landing pads and the irrigated fields beyond. The displays are supposed to make the underground building feel less claustrophobic. "Care for a drink?" he asks.`
 			choice
 				`	"Sure."`
 				`	"No thanks."`
```

Comment:
Just below, the dialogue is significantly dampened now that it doesn't reference Ingot. There's all of 2000 people on New Sahara and nobody has mentioned anything about them.

Reply:
Thoughts on the suggestion I added? (Might need to change stuff further below to line up with the impending sandstorm.)